Marlin/src/module/planner.h

```diff
@@ -376,7 +376,7 @@ class Planner {
 
       FORCE_INLINE static void unskew(float &cx, float &cy, const float &cz) {
         if (WITHIN(cx, X_MIN_POS, X_MAX_POS) && WITHIN(cy, Y_MIN_POS, Y_MAX_POS)) {
-          const float sx = cx + cy * xy_skew_factor + cz * xz_skew_factor,
+          const float sx = cx + cy * xy_skew_factor + cz * (xz_skew_factor - (xy_skew_factor * yz_skew_factor)),
```

Comment:
Nope. See https://github.com/MarlinFirmware/Marlin/pull/8623#discussion_r157253068

Reply:
This is so weird. I didn't make this change in either PR. Something must have happened with the auto merge to somehow regress the change.